icetea/helper/patricia.js

```diff
@@ -122,6 +137,9 @@ exports.save = async ({ block, state, validators, commitKeys }) => {
       },
       (next) => {
         db.put(lastBlockKey, serializer.serialize(persistBlock), next)
+      },
+      (next) => {
+        db.put(txHashesKey, serializer.serialize(txHashes), next)
```

Comment:
Because hash is base 58, could we use customized serializer base58 => Buffer to save some space (available in icetea common).